quick-start-guide/components/SAML2/saml2-web-app-dispatch/src/main/webapp/bookings.jsp

```diff
@@ -0,0 +1,238 @@
+<!--
+~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Is this the correct license ?

Reply:
The year seems to be missing. I'll correct it. Thank you.